src/main/webapp/app/instructor-dashboard/instructor-course-dashboard.component.ts

```diff
@@ -19,16 +24,28 @@ export class InstructorCourseDashboardComponent implements OnInit, OnDestroy {
     predicate: any;
     reverse: any;
     numberOfExercises = 0;
-    rows = [];
     results = [];
-    participations = [];
-    courseScores = [];
+    ratedResultsArrayList = [];
```

Comment:
all these arrays don't have a type

Reply:
And I think you don't need all of these variables, please try to reduce the number of variables